subprojects/libft/meson.build

```diff
@@ -24,14 +24,6 @@ src_files = files([
   'src/ft_isprint.c',
   'src/ft_isspace.c',
   'src/ft_itoa.c',
-  'src/ft_lstadd.c',
```

Comment:
Suggestion: rewrite these with t_list using t_list_conn?

Reply:
Which function would you feel like needing for `t_list_conn`? `add`/`push` and `delone`/`pop` are already implemented using `insert` and `unlink`. `iter` and `map` are kinda useless because C doesn't support higher order/anonymous functions, and `new` isn't implemented because the new implementation doesn't handle allocations anymore.

Although maybe `push` and `pop` wrappers on the `t_list_meta` could be useful. (i.e. they automatically select where the node needs to be inserted/removed).

Side note: You can comment on a range of line changes by first selecting multiple lines using `shift` and then pressing the `+`.

Reply:
Well my thought is that just having *any* example of t_conn_list in action could be usefull,
but yeah, libft does not exactly lend itself to that with the functions you've written so the point is kinda moot.

also, i thought i had multiple lines selected but nope.